### ISSUE TITLE
Make the status screen accessible

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-MATCMilitaryAffidavit#211]
id: select all statuses

Needs to be accessible in a few ways:

* not currently keyboard accessible, can't seem to change my answer after pressing space or enter on an option. I think this is because of the grid, but not exactly sure
* what each button means is not accessible (the actual mean of "yes", "no", etc. can't be figured out by a screen reader without sequentially going through the whole page)
  * IMO it's also not UX accessible, fairly confusing on my first view of the screen
